modules/ROOT/pages/installing-kubernetes.adoc

```diff
@@ -0,0 +1,264 @@
+* Replicator node: if you are using inter-cluster replication using sg-replicate then there will be one designated replicator node whose configuration is different than the rest of the nodes.
+
+The following sections cover the steps to deploy a cluster of regular Sync Gateway nodes.
+The configuration and deployment controller files will differ if you are deploying a cluster of import nodes or a single replicator node.
+
+For example, to deploy a cluster with 4 regular nodes, 1 import node and 1 replicator node you would run the steps below 3 times, each time with a different configuration and deployment controller file.
+
+=== Configuration file
+
+In this section, you will create the Sync Gateway configuration file and store it in a secret location.
+
+. Create a new file called *sgw-config.json* with the following and replace the `server` key with the addressable Couchbase Server hostname.
++
+[source,json]
+----
+{
+  "logging": {
+    "console": {
+      "log_keys": ["*"]
+    }
+  },
+  "databases": {
+    "db": {
+      "server": "http://cb-example-0000.cb-example:8091", // <1>
+      "bucket": "default",
+      "username": "admin", // <2>
+      "password": "password",
+      "users": { "GUEST": { "disabled": false, "admin_channels": ["*"] } },
+      "allow_conflicts": false,
+      "revs_limit": 20
+    }
+  }
+}
+----
+<1> The server key should point to the Couchbase Server cluster.
+This would typically be of the form `http://CB_SERVER_POD.CB_SERVER_SERVICE_NAME:8091`.
+<2> The username/password keys should match what was setup in the <<pre-requisites, pre-requisites>> section when you configured the Sync Gateway RBAC user.
+. You will use a https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/[Kubernetes configMap] to pass the configuration file to Sync Gateway on launch.
+However, since the Sync Gateway contains sensitive information, it is recommended that you create a secret from the config file and pass that to the Sync Gateway.
+Run the following command to create a secret called "sgw-config" from the specified Sync Gateway configuration file (*sgw-config.json*).
++
+[source,console]
+----
+kubectl create secret generic sgw-config --from-file sgw-config.json
+----
++
+If successful, you will see the following.
++
+[source,console]
+----
+secret "sgw-config" created
+----
+
+=== Deployment controller
+
+In this section you will deploy the Sync Gateway cluster with the configuration file that you created above.
+
+You will be using a https://kubernetes.io/docs/concepts/workloads/controllers/deployment/[Kubernetes deployment controller].
+A deployment controller allows you to define the number of Sync Gateway replicas and other parameters.
+
+. Create a new file called *sgw-deployment.yaml* with the following.
++
+[source,yaml]
+----
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: sync-gateway <1>
+spec:
+  replicas: 2 <2>
```

Comment:
If you were to copy paste this file as-is, it would be rejected by Kubernetes due to the number included in brackets.